lib/sportradar/api/nfl/game.rb

```diff
@@ -19,7 +19,11 @@ def initialize(data)
         @utc_offset = data["utc_offset"]
         @weather = data["weather"]
         @clock = data["clock"]
-        @quarter = data["quarter"]
+        if data["quarter"]
+          @quarter = data["quarter"][0]
+          quarter_data = data["quarter"][1].is_a?(Hash) ? [ data["quarter"][1] ] : data["quarter"][1]
+          @quarters = quarter_data&.map { |hash| Sportradar::Api::Nfl::Quarter.new(hash) }
```

Comment:
I need to look at this a little closer, but if I recall correctly the data comes in like this in `play_by_play` calls: 
```
game.response['quarter'].map &:class
=> [String, Hash, Hash, Hash, Hash]
```
Where the quarters would come in as:
```
["current quarter", {Q1...}, {Q2...}, {Q3...}, {Q4...}]
```
During a live game if it was the first quarter, it would come in as 
`['1', {Q1}]`
If it was second quarter `['2', {Q1}, {Q2}]` etc. etc. 

So if you're looking at that quarter array, the `quarter.first` is always most recent quarter in string format, and `quarter.last` is the actual hash data. 

I don't knkow that `data["quarter"][1]` is quite the right way of looking at that